docs/desktop-integration.rst

```diff
@@ -76,24 +76,24 @@ XEmbed style icons will function on desktops that support them with the
 StatusNotifier
 ^^^^^^^^^^^^^^
 
-StatusNotifier style icons will not function without extra permissions as it
-requires talking to a non-hardenend host service. Risks include impersonation of
-other software and exploitation of bugs in the host service such as image decoders.
+StatusNotifier style icons will not function without extra permissions, as they
+require talking to a non-hardened host service. Risks include impersonation of
+other softwares and exploitation of bugs in host services such as image decoders.
 
-At the very minimum to use StatusNotifier you must have the
+To use StatusNotifier, you must at least have the
 ``--talk-name=org.kde.StatusNotifierWatcher`` permission to register an item.
 
 Depending on the exact implementation of StatusNotifier that your application is
-using it may need session bus ownership of ``org.kde.StatusNotifierItem-$PID-$ITEM_ID``.
+using, it may need session bus ownership of ``org.kde.StatusNotifierItem-$PID-$ITEM_ID``.
 
 This permission is problematic in Flatpak as the ``$PID`` value is often the same
-in sandboxes and the item will possibly conflict with other applications.
-However if needed the ``--own-name=org.kde.*`` permission will allow this. This opens
-many new risks including the ability to impersonate any KDE service or application
+in sandboxes, leading to potential conflicts with other applications.
+However, if needed, the ``--own-name=org.kde.*`` permission will allow this. This introduces
+many new risks, including the ability to impersonate any KDE service or application,
 possibly capturing important user data.
```

Comment:
can you please make a separate commit to turn the last sentence into a warning label ?

Reply:
like so https://github.com/flatpak/flatpak-docs/blob/d3313532e7bd1c8aab1817205820df6a5cd3ad55/docs/single-file-bundles.rst?plain=1#L10